packages/flutter_tools/lib/src/migrate/migrate_utils.dart

```diff
@@ -0,0 +1,325 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
```

Comment:
importing `dart:io` is banned from the tool. Instead, use https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/base/io.dart